cuppa/src/main/java/org/forgerock/cuppa/model/Options.java

```diff
@@ -75,14 +65,29 @@ public static Options copyOf(Options options) {
     }
 
     /**
+     * Removes an option.
+     *
+     * @param optionClass The class of the option to remove.
+     * @param <O> The type of the option.
+     * @return A new Option instance without the specified option.
+     */
+    public <O extends Option<?>> Options remove(Class<O> optionClass) {
```

Comment:
One final comment (sorry): I think `unset` is probably a better name considering the other method is called `set`.